app/routes/authentication.py

```diff
@@ -61,6 +65,23 @@ def login():
     return jsonify(ret), 200
 
 
+@auth_blueprint.route("/admin", methods=["POST"])
+@jwt_required
+def admin_login():
```

Comment:
what does this path do?

Reply:
I was thinking that when the user wanted to change something like their email/password, it would prompt them to verify their credentials (via this admin_login route).  

Then once verified, give them a short-lived fresh token.  

The fresh token gives them access to update their credentials. 
Since `[PUT] /user/<id>` has the decorator `@fresh_jwt_required`, only those with fresh tokens can do that operation.